.github/workflows/localize.yml

```diff
@@ -0,0 +1,34 @@
+name: Localize
+
+on: 
+  push:
+    branches:
+      - master
```

Comment:
could we make this to be only on changes for xib or .string files later?

Reply:
Not that I'm aware of. See https://help.github.com/en/actions/reference/events-that-trigger-workflows for a list of ways you can invoke. Since we won't know what files changed until we've actually checked the commit, and we can only check the contents of the commit when we're in the workflow, I don't think we can avoid running the workflow.

Luckily, this is in line with how the API is designed to be called, and no PR will be made if no changes are detected (which will be true for the majority of runs of this pipeline).